test/lib/Launcher.ts

```diff
@@ -129,7 +129,8 @@ export class Launcher {
             .use((req, res) => {
                 res.status(200).send("OK");
 
-                eventBus.publish(new DataEvent(req.body.trigger, req.body.data));
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                eventBus.publish(new DataEvent(req.body.trigger, { ...req.body.data, _headers: req.headers }));
```

Comment:
I didn't meant to write the headers to the event data. Just add a new ConnectorDataEvent that extends DataEvent and add headers.